liquibase/src/main/java/io/micronaut/liquibase/LiquibaseConfigurationProperties.java

```diff
@@ -140,22 +112,6 @@ public boolean isTestRollbackOnUpdate() {
         return testRollbackOnUpdate;
     }
 
-    /**
-     * @return true if classpath prefix should be ignored during changeset comparison
-     */
-    public boolean isIgnoreClasspathPrefix() {
-        return ignoreClasspathPrefix;
-    }
-
-    /**
-     * Ignores classpath prefix during changeset comparison. Default value ({@value #DEFAULT_IGNORECLASSPATHPREFIX}).
-     *
-     * @param ignoreClasspathPrefix Sets whether to ignore the classpath prefix during changeset comparison.
-     */
-    public void setIgnoreClasspathPrefix(boolean ignoreClasspathPrefix) {
-        this.ignoreClasspathPrefix = ignoreClasspathPrefix;
-    }
-
```

Comment:
These methods cannot be removed if you want this included in Micronaut 2. You can change the implementation to log a warning saying this configuration option is no longer necessary and to remove it and deprecate the methods.

Reply:
Done